package.json

```diff
@@ -10,29 +10,29 @@
     "test": "make test"
   },
   "devDependencies": {
-    "@babel/cli": "7.0.0-beta.52",
-    "@babel/core": "7.0.0-beta.52",
-    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.52",
-    "@babel/preset-env": "7.0.0-beta.52",
-    "@babel/preset-flow": "7.0.0-beta.52",
-    "@babel/preset-stage-0": "7.0.0-beta.52",
-    "@babel/register": "7.0.0-beta.52",
+    "@babel/cli": "^7.0.0-beta.54",
```

Comment:
Thanks for the PR, we aren't using `^` for beta's. This is something we recommend against for user's of Babel, and we are in that group 😁

Reply:
gotcha. I'll fix that.